### ISSUE TITLE
Adjust names of templates

### ISSUE THREAD

**Issue by jenniferward** [rism-digital/muscat#1559]
Along with https://github.com/rism-digital/muscat/issues/1558, we'd like to adjust the names of the templates so that they are grouped together better and so that the information given in the titles is consistent. This affects the drop-down list in Sources 

![image](https://github.com/rism-digital/muscat/assets/18663291/0e8d3502-2183-424f-882a-ef3e2f5edd70)

as well as the names of the templates that appear when creating a new record: 

![image](https://github.com/rism-digital/muscat/assets/18663291/e2f201a9-2171-4570-a15a-a34000eccc28)

We suggest the following:
Manuscript collection: Notated music, libretto/song text, treatise
Manuscript item in a collection or single item: Notated music
Manuscript item in a collection or single item: Libretto/song text
Manuscript item in a collection or single item: Treatise
Printed collection or single item: Notated music
Printed item in a collection: Notated music
Printed collection or single item: Libretto/Song text
Printed item in a collection: Libretto/song text
Printed collection or single item: Treatise
Printed item in a collection: Treatise
Composite volume

I'm not sure if space is an issue, but can we try this out?

**Comment by xhero** [comment]
I was waiting for #1558 before doing this, should I still go ahead?

**Comment by jenniferward** [comment]
Yes, we need to do both at the same time to make sure it looks good optically and coordinate the languages.
